src/components/EventList.tsx

```diff
@@ -30,7 +31,9 @@ const EventList = ({ events, org } : EventListProps) : JSX.Element => {
                         <View data-test="start-time">{ e.start_time }</View>
                         <View data-test="end-time">{ e.end_time }</View>
                         <View data-test="location-title">{ e.location.title }</View>
-                        <Button data-test="sign-up-button" variant="cta">Sign-up</Button>
+                        <Button data-test="sign-up-button" variant="cta">
+                            <Msg id="components.eventList.signup_button"/>
```

Comment:
This line is mixing `camelCase` and `snake_case` (and this happens on multiple occasions in this PR). I think we should stick to `camelCase` for localized messages.